sonar-unit-tests.xsd

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Would be great if you could add a test to the current tests that generates the file and validates it, pretty sure there is an npm module to do that :)